crates/node/service/src/actors/engine.rs

```diff
@@ -104,6 +119,20 @@ impl NodeActor for EngineActor {
                         warn!(target: "engine", "Encountered error draining engine api tasks: {:?}", e);
                     }
                 }
+                attributes = self.attributes_rx.recv() => {
+                    let Some(attributes) = attributes else {
+                        debug!(target: "engine", "Received `None` attributes from receiver");
+                        continue;
+                    };
+                    let task = BuildTask::new(
+                        Arc::clone(&self.client),
+                        Arc::clone(&self.config),
+                        attributes,
+                        true,
+                    );
+                    let task = EngineTask::BuildBlock(task);
```

Comment:
We'll need to add another check here that does the L1 consolidation. We only want to build a block if `safe` + `unsafe` are equal.

If you'd like to get this in and iterate tho all for it.

Reply:
Yea the whole point of this stack is just to get the basic channels set up between actors. Then the correct rules can be applied.